launch/utils.py

```diff
@@ -7,3 +7,45 @@ def trim_kwargs(kwargs_dict: Dict[Any, Any]):
     """
     dict_copy = {k: v for k, v in kwargs_dict.items() if v is not None}
     return dict_copy
+
+
+def infer_env_params(env_selector: Optional[str]):
+    """
+    Returns an env_params dict from the env_selector.
+
+    env_selector: str - Either "pytorch" or "tensorflow"
+    """
+    if env_selector == "pytorch":
+        import torch
+
+        try:
+            ver = torch.__version__.split("+")
+            torch_version = ver[0]
+            cuda_version = ver[1][2:] if len(ver) > 1 else "113"
```

Comment:
can also default to "latest" if parsing goes awry but i'm hesitant cuz upstream build issues are worse than client side errors